argoverse/evaluation/competition_util.py

```diff
@@ -169,6 +169,7 @@ def dist(p1: Tuple[float, float], p2: Tuple[float, float]) -> float:
 
 
 def poly_to_label(poly: Polygon, category: str = "VEHICLE", track_id: str = "") -> ObjectLabelRecord:
+    """ """
```

Comment:
Thanks for taking care of this, @johnwlambert. Could we add a short docstring here?

Reply:
Sure -- just added one